tests/components/airvisual/test_config_flow.py

```diff
@@ -42,6 +53,47 @@
         assert result["errors"] == {CONF_API_KEY: "invalid_api_key"}
 
 
+async def test_migration_1_2(hass):
+    """Test migrating from version 1 to version 2."""
+    conf = {
+        CONF_API_KEY: "abcde12345",
+        CONF_GEOGRAPHIES: [
+            {CONF_LATITUDE: 51.528308, CONF_LONGITUDE: -0.3817765},
+            {CONF_LATITUDE: 35.48847, CONF_LONGITUDE: 137.5263065},
+        ],
+    }
+
+    config_entry = MockConfigEntry(
+        domain=DOMAIN, version=1, unique_id="abcde12345", data=conf
+    )
+    config_entry.add_to_hass(hass)
+
+    assert len(hass.config_entries.async_entries(DOMAIN)) == 1
+
+    with patch("pyairvisual.api.API.nearest_city"):
+        assert await async_setup_component(hass, DOMAIN, {DOMAIN: conf})
+
+    config_entries = hass.config_entries.async_entries(DOMAIN)
+
+    assert len(config_entries) == 2
+
+    assert config_entries[0].unique_id == "51.528308, -0.3817765"
+    assert config_entries[0].title == "Cloud API (51.528308, -0.3817765)"
+    assert config_entries[0].data == {
+        CONF_API_KEY: "abcde12345",
+        CONF_LATITUDE: 51.528308,
+        CONF_LONGITUDE: -0.3817765,
+    }
+
+    assert config_entries[1].unique_id == "35.48847, 137.5263065"
+    assert config_entries[1].title == "Cloud API (35.48847, 137.5263065)"
+    assert config_entries[1].data == {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5238698)